chanzuckerberg/idseq-web
# Allow operator to deposit custom file on user Downloads page

# Description
This is intended as a mechanism for an operator to securely expose a file to a user for download via a rake task. The file appears as a Bulk Download of type "Customer Support Request".

![image](https://user-images.githubusercontent.com/16944816/80769762-fcb5b000-8ae9-11ea-95da-ac19c4d77bcf.png)

# Tests
- Verify that running the rake task allows successful addition of a file to the Downloads page for only the intended user.
- Verify that the description of "Customer Support Request" type downloads appears in the Details sidebar on the Downloads page.
- Verify that all previously existing Bulk Download types work exactly the same as before.

# Tests for QA
- Verify that all previously existing Bulk Download types work exactly the same as before.
